packages/slides-frontend/src/containers/Editor/components/image/Upload/Dropzone.js

```diff
@@ -117,7 +117,7 @@ Dropzone.propTypes = {
 function mapDispatchToProps(dispatch) {
   return {
     onImageRequest: () => dispatch(uploadImageRequest(false)),
-    onAddImage: src => dispatch(addItem({type: ItemTypes.IMAGE, src})),
+    onAddImage: Src => dispatch(addItem({type: ItemTypes.IMAGE, Src})),
```

Comment:
Why this change? Does it really have to be uppercased?

Reply:
Yes, because in the reducer of the DeckReducer  I can get the `Src` attribute of the object and assign it directly to the Image object. Now everywhere that the src of the image is referenced the `Src` keyword is used.

Reply:
Yeah, I saw it. I am just not sure why this even has to be uppercase in the class where it is used :P

Reply:
hm more about consistency I guess, should I change it everywhere?
Also the Position, Size, Data, Edit, ID ?

Reply:
but okay I can change only the Src if you want

Reply:
no, no, you don't have to change it :) I was more curious why it started like that ;) But let's keep it as it is.

Reply:
And what I meant is that you could have something like

```javascript
onAddImage: src => dispatch(addItem({type: ItemTypes.IMAGE, Src: src})),
```

Reply:
ah yes okay! :smile: 
So I leave it as is for now.